pkg/apis/scorecard/v1alpha3/test_types.go

```diff
@@ -28,6 +28,8 @@ type TestResult struct {
 	Errors []string `json:"errors,omitempty"`
 	// Suggestions is a list of suggestions for the user to improve their score (if applicable)
 	Suggestions []string `json:"suggestions,omitempty"`
+	// Timestamp of the result from an invidual test
+	Tstamp metav1.Time `json:"tstamp,omitempty"`
```

Comment:
Since you mentioned the timestamp would be when the result is created, wdyt about a more descriptive name?

Also GoDoc comments should always start with the name of the identifier it is describing.
```suggestion
	// CreationTimestamp of the result from an invidual test
	CreationTimestamp metav1.Time `json:"creationTimestamp,omitempty"`
```

Reply:
That's fine with me, the x-unit field is assigned from this value but doesn't care what it's called, this more descriptive name will keep the API cleaner going forward.